test/CalendarDate.test.ts

```diff
@@ -1192,4 +1193,24 @@ describe('CalendarDate', () => {
       );
     });
   });
+
+  describe('Test of instanceOf', () => {
+    test('Returns true for instanceOf CalendarDate', () => {
+      fc.assert(
+        fc.property(
+          fc.integer({ min: 200, max: 9995 }),
+          fc.integer({ min: 1, max: 12 }),
+          fc.integer({ min: 1, max: 31 }),
+          (year, month, day) => {
+            // Arrange
+            day = ensureValidDay(year, month, day);
+            const calendarDate = new CalendarDate(year, month, day);
+
+            // Assert
+            expect(instanceOf(calendarDate, CalendarDate)).toBeTruthy();
```

Comment:
I would just test if the function works and returns the CalendarDate string. I don't see why we should test against a graphql utility function here, since calendar-date is not related to graphql. You can also remove the fc all together for this test since it has nothing to do with the actual values of the calendar date object